platform/broadcom/sonic-platform-modules-accton/as5835-54x/utils/accton_as5835_54x_util.py

```diff
@@ -364,6 +369,73 @@ def system_ready():
         return False
     return True
 
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
+PLATFORM_API2_WHL_FILE_PY2 ='sonic_platform-1.0-py2-none-any.whl'
+PLATFORM_API2_WHL_FILE_PY3 ='sonic_platform-1.0-py3-none-any.whl'
+def do_sonic_platform_install():
+    device_path = "{}{}{}{}".format(PLATFORM_ROOT_PATH, '/x86_64-accton_', PROJECT_NAME, '-r0')
+    SONIC_PLATFORM_BSP_WHL_PKG_PY2 = "/".join([device_path, PLATFORM_API2_WHL_FILE_PY2])
+    SONIC_PLATFORM_BSP_WHL_PKG_PY3 = "/".join([device_path, PLATFORM_API2_WHL_FILE_PY3])
```

Comment:
All applications which import sonic_platform should now support Python 3 (as of a couple weeks ago). Therefore, I believe there should no longer be a need to build/install the Python 2 package. Would you mind confirming, and if so, remove all references to the Python 2 package in this PR?

Reply:
OK.  I will git pull new code base to check it.